v19.1/architecture/life-of-a-distributed-transaction.md

```diff
@@ -129,14 +129,10 @@ If an operation encounters a write intent for a key, it attempts to "resolve" th
 
 - `COMMITTED`, this operation converts the write intent to a regular key-value pair, and then proceeds as if it had read that value instead of a write intent.
 - `ABORTED`, this operation discards the write intent and reads the next-most-recent value from RocksDB.
-- `PENDING`, the new transaction attempts to "push" the write intent's transaction by moving that transaction's timestamp forward (i.e., ahead of this transaction's timestamp); however, this only succeeds if the write intent's transaction has become inactive.
-
-  If the push succeeds, the operation continues.
-
-  If this push fails (which is the majority of the time), this transaction goes into the [`TxnWaitQueue`](https://www.cockroachlabs.com/docs/stable/architecture/transaction-layer.html#txnwaitqueue) on this node. The incoming transaction can only continue once the blocking transaction completes (i.e., commits or aborts).
-- _Missing_, the resolver consults the write intent's timestamp. If it was created within the transaction liveness threshold, it treats the transaction record as exhibiting the `PENDING` behavior, with the addition of tracking the push in the range's timestamp cache, which will inform the transaction that its timestamp was pushed once the transaction record gets created.
-
-    If the write intent is older than the transaction liveness threshold, the resolution exhibits the `ABORTED` behavior. 
+- `PENDING`, the new transaction attempts to "push" the write intent's transaction by moving that transaction's timestamp forward (i.e.,  ahead of this transaction's timestamp); however, this only succeeds if the write intent's transaction has become inactive.
+  	- If the push succeeds, the operation continues.
+  	- If this push fails (which is the majority of the time), this transaction goes into the [`TxnWaitQueue`](https:www.cockroachlabs.com/docs/stable/architecture/transaction-layer.html#txnwaitqueue) on this node. The incoming transaction can only continue once the blocking transaction completes (i.e., commits or aborts).
+- `MISSING`, the resolver consults the write intent's timestamp. If it was created within the transaction liveness threshold, it treats the transaction record as exhibiting the `PENDING` behavior, with the addition of tracking the push in the range's timestamp cache, which will inform the transaction that its timestamp was pushed once the transaction record gets created. If the write intent is older than the transaction liveness threshold, the resolution exhibits the `ABORTED` behavior. 
```

Comment:
1.Should we write "If it was created within the transaction liveness threshold,....." and "If the write intent is older than the transaction liveness threshold, the resolution exhibits the `ABORTED` behavior. " as indented bullets?

2. I read the STAGING option at https://www.cockroachlabs.com/docs/stable/architecture/transaction-layer.html, should we include that here too?

The above 2 points apply to all four files that I have updated for this issue.